src/main/java/org/opentripplanner/netex/loader/mapping/RouteMapper.java

```diff
@@ -76,10 +83,21 @@ private Agency findOrCreateAgency(Line line) {
         // No authority found in Network or GroupOfLines.
         // Use the default agency, create if necessary
         LOG.warn("No authority found for " + line.getId());
-        Agency agency = agencyMapper.createDefaultAgency();
-        if (!agenciesById.containsKey(agency.getId())) {
+
+        Agency agency = agenciesById.get(authorityMapper.dummyAgencyId());
```

Comment:
Normalize terminology: dummy vs. default agency. Clarify that this is really for missing agencies (not specified in input data).